Add React 19 to peerDep range

React 19 is now Stable.
```

npm error code ERESOLVE
npm error ERESOLVE unable to resolve dependency tree
npm error
npm error Found: react@19.0.0
npm error node_modules/react
npm error     @package@0.0.0
npm error
npm error Could not resolve dependency:
npm error peer react@"0.14 - 18" from react-signature-canvas@1.0.6
npm error node_modules/react-signature-canvas
npm error   react-signature-canvas@"^1.0.6"
```

--- a/package.json
+++ b/package.json
@@ -89,7 +89,7 @@
     "@rollup/plugin-babel": "^5.3.1",
     "@rollup/plugin-commonjs": "^21.1.0",
     "@rollup/plugin-node-resolve": "^13.2.1",
-    "@testing-library/react": "^13.3.0",
+    "@testing-library/react": "^16.1.0",
     "canvas": "^3.0.0",
     "concurrently": "^7.1.0",
     "jest": "^27.5.1",

--- a/package.json
+++ b/package.json
@@ -65,13 +65,13 @@
   },
   "peerDependencies": {
     "prop-types": "^15.5.8",
-    "react": "0.14 - 18",
-    "react-dom": "0.14 - 18"
+    "react": "0.14 - 19",
+    "react-dom": "0.14 - 19"
   },
   "dependencies": {
     "@babel/runtime": "^7.17.9",
     "@types/prop-types": "^15.7.3",
-    "@types/react": "^18.0.12",
+    "@types/react": "^19.0.2",
     "@types/signature_pad": "^2.3.0",
     "signature_pad": "^2.3.2",
     "trim-canvas": "^0.1.0"
@@ -96,8 +96,8 @@
     "jest-config": "^27.5.1",
     "package-json-type": "^1.0.3",
     "parcel": "^2.4.1",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
+    "react": "^19.0.0",
+    "react-dom": "^19.0.0",
     "rollup": "^2.70.2",
     "rollup-plugin-node-externals": "^4.0.0",
     "rollup-plugin-terser": "^7.0.2",

--- a/src/index.tsx
+++ b/src/index.tsx
@@ -9,7 +9,7 @@ export interface SignatureCanvasProps extends SignaturePad.SignaturePadOptions {
 }
 
 export class SignatureCanvas extends Component<SignatureCanvasProps> {
-  static propTypes = {
+  static override propTypes = {
     // signature_pad's props
     velocityFilterWeight: PropTypes.number,
     minWidth: PropTypes.number,

--- a/test/index.spec.tsx
+++ b/test/index.spec.tsx
@@ -5,7 +5,7 @@ import React from 'react'
 import { SignatureCanvas, SignatureCanvasProps } from '../src/index'
 import { propsF, dotF } from './fixtures'
 
-function renderSCWithRef (props?: SignatureCanvasProps): { wrapper: RenderResult, instance: SignatureCanvas, ref: React.RefObject<SignatureCanvas> } {
+function renderSCWithRef (props?: SignatureCanvasProps): { wrapper: RenderResult, instance: SignatureCanvas, ref: React.RefObject<SignatureCanvas | null> } {
   const ref = React.createRef<SignatureCanvas>()
   const wrapper = render(<SignatureCanvas {...props} ref={ref} />)
   const instance = ref.current! // eslint-disable-line @typescript-eslint/no-non-null-assertion -- this simplifies the code; it does exist immediately after render. it won't exist after unmount, but we literally test for that separately

